core/src/main/resources/jenkins/views/JenkinsHeader/headerContent.jelly

```diff
@@ -1,6 +1,6 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:i="jelly:fmt" xmlns:x="jelly:xml">
-    <header id="header" class="page-header">
+    <header id="page-header" class="page-header">
```

Comment:
This looks like it could easily break some stuff inadvertently for no real reason?